src/site/mod.rs

```diff
@@ -8,6 +8,9 @@ mod head;
 mod header;
 mod html;
 pub mod markdown;
+extern crate fs_extra;
+use fs_extra::copy_items;
+use fs_extra::dir;
```

Comment:
i would not bring these into scope because it will be confusing where they come from

Reply:
also fixed